lisa/sut_orchestrator/azure/features.py

```diff
@@ -548,8 +548,7 @@ def create_setting(
     ) -> Optional[schema.FeatureSettings]:
         raw_capabilities: Any = kwargs.get("raw_capabilities")
 
-        value = raw_capabilities.get("RdmaEnabled", None)
-        if value and eval(value) is True:
+        if raw_capabilities.get("RdmaEnabled", None) == "True":
```

Comment:
I don't remember its value, may be "true"?

Reply:
The only value it could be with the existing code is `'True'`. `'true'` would raise a `NameError`. Typically this would all run through a data validation library, but that's out of scope for this PR.

Reply:
it is False or True